Source/WebKit/Scripts/webkit/parser.py

```diff
@@ -86,13 +86,14 @@ def parse(file):
                 parameters = []
 
             runtime_enablement = None
+            enabled_by = None
```

Comment:
Can we rename `runtime_enablement` to `enabled_if` for clarity?

Reply:
Sure, we can do that.